stages/org.osbuild.anaconda.interactive-defaults

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python3
+"""
+Create an Anaconda interactive defaults file
+
+Interactive defaults files are a way to program
+the Anaconda installer.  This stage can be used
+to create such an interactive defaults file at
+the location expected by anaconda.
+
+Only a very limited set of available kickstart
+commands are supported here.
+"""
+
+import os
+import sys
+
+import osbuild.api
+
+SCHEMA = """
+"additionalProperties": false,
+"properties": {
+  "liveimg": {
+    "type": "object",
+    "required": ["url"],
+    "properties": {
+      "url": {
+        "type": "string"
+      }
+    }
+  }
+}
+"""
+
+
+def main(tree, options):
+    path = "usr/share/anaconda/interactive-defaults.ks"
+
+    config = []
+
+    liveimg = options.get("liveimg")
+    if liveimg:
+        url = liveimg["url"]
+        config += [f"liveimg --url {url}"]
+
+    target = os.path.join(tree, path)
+    base = os.path.dirname(target)
+    os.makedirs(base, exist_ok=True)
+
+    with open(target, "w", encoding="utf8") as f:
+        if config:
+            f.write("\n".join(config))
+        f.write("\n")
+
+    print(f"created interactive defaults at: {path}\n")
+    with open(target, "r", encoding="utf8") as f:
+        print(f.read())
```

Comment:
Any particular reason to include this? I don't mind it, but we don't do this often, do we?

Reply:
It comes from the kickstart stage that this one was based on so I left it in; on second thought, it's not necessary :)